enzy_htp/structure/structure.py

```diff
@@ -389,14 +429,14 @@ def __eq__(self, other: Structure) -> bool:
                     #              residues are relatively conserved, but there are exceptions, e.g. different ligand both name LIG)
 
     def __ne__(self, other: Structure) -> bool:
-        """Negation operator for other Structure() objects. Inverstion of Structure.__eq__()."""
+        '''Negation operator for other Structure() objects. Inverstion of Structure.__eq__().'''
         return not (self == other)
     #endregion
 
     #region (TODO+OLD)
     # === TODO ===
     def get_connect(self, metal_fix=1, ligand_fix=1, prepi_path=None): #@shaoqz: @nu ### TODO rewrite doc # also change a name this is not a getter but to generate sth.
-        """
+        '''
```

Comment:
I do not think this docstring complies with google style guide for pythong

Reply:
Right. This is an old method under the TODO section. I will address these TODO code along with updating the refactored Structure to mutation, preparation, etc.

Reply:
The change here was because I globally changed double quotes to single for this file lol